pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.34.1
+version: 1.34.2
```

Comment:
Should we mark this as a dev version until #1347 is fixed as well?

Reply:
Yeah, that makes sense. There's a decent chance the other bug is also causing issues with the internal migration; I just haven't noticed it because I've been focused on this one